src/AppKit/NSFont.cs

```diff
@@ -174,9 +172,7 @@ public static NSFont ControlContentFontOfSize (nfloat fontSize)
 #if NET
 		[SupportedOSPlatform ("macos")]
 		[UnsupportedOSPlatform ("macos10.13")]
-#if MONOMAC
-		[Obsolete ("Starting with macos10.13.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
-#endif
+		[ObsoletedOSPlatform ("macos10.13.")]
```

Comment:
This is probably an invalid version number (and this happens in a few more places in this PR as well).

Reply:
Arg, I thought I got them all (there was a bug in my macro).

I'll find them all shortly.

Reply:
woah nice catch 👀